spec/models/section_spec.rb

```diff
@@ -3,9 +3,15 @@
 describe Section do
   let(:instance) { described_class.new }
   let(:date) { DateTime.now }
+  let(:monograph) { create(:monograph) }
```

Comment:
You don't need to actually create a monograph here.  It will slow down the tests to create objects in fedora.  Instead you can do something like this:
```ruby
let(:monograph_id) { '123' }

it "has a monograph_id" do
  instance.monograph_id = monograph_id
  expect(instance.monograph_id).to eq monograph_id
end
```